### ISSUE TITLE
Empty-args error triggered by functions

### ISSUE THREAD

**Issue by mirisuzanne** [sasstools/sass-lint#862]
I'm getting this error:

```
69:18  error  Parenthesis should be removed  empty-args
```

when I call the `susy-gutter()` function without arguments, using sass-lint `1.9.1`. I would expect this to only error on mixins, since the empty parenthesis are required for a function call.

Here's the full surrounding code block (using Susy and True):

```scss
    @include assert-equal(
      susy-gutter(),
      5%,
      'Gutters are calculated as a percentage of context');
```

**Comment by DanPurdy** [comment]
That's odd, it looks like it's because it's wrapped within an include itself as the rule is looking for argument nodes within include or mixin nodes. I'll look into it. Thanks for the report. 👍

**Comment by littlstonee** [comment]
@DanPurdy Any fix for this bug? It seems the bug still exists.
